Source/WebCore/platform/graphics/FontDescription.h

```diff
@@ -145,7 +145,7 @@ class FontDescription {
     void setShouldDisableLigaturesForSpacing(bool shouldDisableLigaturesForSpacing) { m_shouldDisableLigaturesForSpacing = shouldDisableLigaturesForSpacing; }
     void setFontPalette(const FontPalette& fontPalette) { m_fontPalette = fontPalette; }
     void setFontSizeAdjust(FontSizeAdjust fontSizeAdjust) { m_sizeAdjust = fontSizeAdjust; }
-    void setEnableEvaluationTimeZoom(bool enableEvaluationTimeZoom) { m_enableEvaluationTimeZoom = enableEvaluationTimeZoom; }
+    void setEvaluationTimeZoom(bool evaluationTimeZoom) { m_evaluationTimeZoom = evaluationTimeZoom; }
```

Comment:
We sometimes use a `bool = true` to be able to write `foo.setEvaluationTimeZoom()`  which reads nicely